subsystem/kubeapi_test.go

```diff
@@ -2193,6 +2284,10 @@ var _ = Describe("[kube-api]cluster installation", func() {
 		checkAgentCondition(ctx, day2Host2.ID.String(), v1beta1.ValidatedCondition, v1beta1.ValidationsPassingReason)
 	})
 
+	It("deploy clusterDeployment with networkType configuration", func() {
+
+	})
```

Comment:
@slaviered did you mean to add some test content here?

Reply:
No. I will remove this